### PR TITLE
1510 dont show unregistered schemes

### PR DESCRIPTION
**Pull request by tibetsprague**
Fixes https://github.com/daostack/alchemy/issues/1510

### CODE REVIEW

**Comment by tibetsprague** [comment]
Depends on https://github.com/daostack/client/pull/435

**Comment by orenyodfat** [comment]
review app fail in dxdao .
see https://alchemy-1510-dont-show-fil2qbs.herokuapp.com/dao/0x519b70055af55a007110b4ff99b0ea33071c720a which is now point to mainnet  @tibetsprague

**Comment by tibetsprague** [comment]
yeah this won't work until the new client is available

File: package.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
     "postinstall": "rimraf ./src/**/*.scss.d.ts",
     "service-status": "node scripts/serviceStatus.js",
     "start": "cross-env NODE_ENV=development NETWORK=private node --max_old_space_size=4096 ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --config webpack.dev.config.js",
-    "start-dev-profiling": "cross-env NODE_ENV=dev-profiling NETWORK=rinkeby API_URL=https://daostack-alchemy.herokuapp.com S3_BUCKET=daostack-alchemy-staging webpack-dev-server --config webpack.dev.config.js",
+    "start-dev-profiling": "cross-env NODE_ENV=dev-profiling NETWORK=rinkeby webpack-dev-server --config webpack.dev.config.js",
```
Comment (orenyodfat) [inline package.json]: why this change ?

**Comment by orenyodfat** [comment]
this [workaround](https://github.com/daostack/alchemy/pull/1513/files) need to be removed

File: package.json (hunk 1 above)
Comment (tibetsprague) [reply package.json]: we dont use API_URL anymore

Comment (orenyodfat) [reply package.json]: is that related to this pr ?

Comment (tibetsprague) [reply package.json]: well no. but it was such a small thing i bundled it in :)
